src/status_im/ui/screens/profile/styles.cljs

```diff
@@ -4,7 +4,7 @@
             [status-im.utils.platform :as platform])
   (:require-macros [status-im.utils.styles :refer [defstyle]]))
 
-(defstyle toolbar-edit-text
+(defstyle toolbar-ios-text
```

Comment:
Is is only for iOS?

Reply:
Yes, in new designs android uses icons for actions in toolbars, while ios uses labels, e.g. checkmark vs Done